lua/complementree/sources.lua

```diff
@@ -162,17 +165,80 @@ local function lsp_completedone(completed_item)
   end
 end
 
+M.create_filepath_matches = function(opts)
```

Comment:
I would really like this like so:
```suggestion
M.filepath_matches = function(opts)
```